### PR TITLE
Allow specification of `enable_flow_logs`

### PR DESCRIPTION
**Pull request by smaslennikov**

### CODE REVIEW

File: modules/gcp/v0/1/colossus_layer/main.tf [hunk 1]
```diff
@@ -18,6 +18,7 @@ variable "network_cidr_block" {}
 variable "network_name_fancy" {}
 variable "region" {}
 variable "vpc_id" {}
+variable "enable_flow_logs" {}
```
Comment (naftulikay) [inline modules/gcp/v0/1/colossus_layer/main.tf]: set `default = false` just for sanity's sake, otherwise LGTM

Comment (smaslennikov) [reply modules/gcp/v0/1/colossus_layer/main.tf]: even squashed 4 u

**Comment by smaslennikov** [comment]
Hah, now we're hitting Terraform AWS provider deprecations.

**Comment by naftulikay** [comment]
Let me see if I can fix this.

**Pull request merged**